### PR TITLE
Reorganize ffhw and fflev with delegation

### PR DESCRIPTION
**Pull request by kevin-olbrich**

### CODE REVIEW

File: hueckeswagen [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - kevin@vfn-nrw.de
+networks:
+  ipv4:
+    - 10.146.0.0/16 #Hückeswagen
+  ipv6:
+    - 2001:bf7:100:2192::/64
+    - fddf:ebfd:a801:2192::/64
+domains:
+  - ffhw
+delgate:
```
Comment (mweinelt) [inline hueckeswagen]: `delegate`

File: leverkusen [hunk 2]
```diff
@@ -0,0 +1,17 @@
+tech-c:
+  - felix@vfn-nrw.de
+  - ruben@vfn-nrw.de
+networks:
+  ipv4:
+    - 10.76.0.0/16 #Leverkusen
+  ipv6:
+    - 2001:bf7:100:2144::/64
+    - fddf:ebfd:a801:2144::/64
+delgate:
```
Comment (mweinelt) [inline leverkusen]: `delgate`

File: hueckeswagen [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  65530:  
+    ipv4:
```
Comment (mweinelt) [inline hueckeswagen]: drop the ipv4/ipv6 blocks and put all subnets on the same level as a list below the asn:
```
  65530:
    - 10.146.0.0/16
    - 2001:bf7:100:2192::/64
  ....
```

**Comment by kevin-olbrich** [comment]
Thank you @mweinelt.
How do I fix "Delegated Network ... is not contained in any existing network"?

**Comment by mweinelt** [comment]
This was an issue in the check script, where it would only allow longer prefixes than the allocated network to get delegated.

Fixed in https://github.com/freifunk/icvpn-scripts/commit/93ce3e35ccf9e8a29f6dec62afaaa72e17c65846

**Pull request merged**